File: src/api/openness.ts
import { get } from '@/utils/request'

export function getLoginConfig<T>() {
  return get<T>({
    url: '/openness/loginConfig',
  })
}

// 获取免责声明
export function getDisclaimer<T>() {
  return get<T>({
    url: '/openness/getDisclaimer',
  })
}

// 获取关于的描述信息
export function getAboutDescription<T>() {
  return get<T>({
    url: '/openness/getAboutDescription',
  })
}

File: service/router/openness/openness.go
package openness

import (
	"sun-panel/api/api_v1"

	"github.com/gin-gonic/gin"
)

func InitOpenness(router *gin.RouterGroup) {
	api := api_v1.ApiGroupApp.ApiOpen.Openness
	{
		router.GET("loginConfig", api.LoginConfig)
		router.GET("getDisclaimer", api.GetDisclaimer)
		router.GET("getAboutDescription", api.GetAboutDescription)
	}
}
